project1/css/index.css

```diff
@@ -24,19 +24,23 @@ time, mark, audio, video {
 	vertical-align: baseline;
 }
 .container{
-	width: 900px;
-	margin:0 auto;
+	
+	width: 1000px;
+	margin: 0 auto;
 
 }
 header{
-	width: 100%;
+display:flex;
```

Comment:
The display: flex property isn't needed here, and it could cause problems later on.  I would go ahead and remove it.